packages/calcite-components/src/components/label/usage/Basic.md

```diff
@@ -1,7 +1,12 @@
 ```html
 <calcite-label>
-  Invalid input
-  <calcite-input status="invalid" type="search" placeholder="Filter your files" value="adfo2h2"></calcite-input>
-  <calcite-input-message icon status="invalid"> Something doesn't look right </calcite-input-message>
+  Email address
+  <calcite-input
+    status="invalid"
+    type="email"
+    placeholder="Enter your email address"
+    value="invalidemailformat.com"
+  ></calcite-input>
+  <calcite-input-message icon status="invalid"> Enter a valid email address </calcite-input-message>
```

Comment:
nitpick:
For readability, could the spaces at the beginning and tail end be removed?